src/FSAL/FSAL_CORTXFS/conf/nfs_setup.sh

```diff
@@ -303,6 +297,65 @@ function cortx_nfs_config {
 	echo -e "\nNFS setup is complete"
 }
 
+function cortx_ganesha_restart {
+	echo -e "\nStopping NFS Ganesha"
+	systemctl stop nfs-ganesha || die "Failed to stop NFS-Ganesha"
+
+	echo -e "\nUpdating conf file"
```

Comment:
Instead of going for dummy export right-away, should we check if really need the dummy export ?
Or else we should always have dummy export ,  not visible to anyone ?

Reply:
an export block is read and if there are any issues that export block is deleted. So we simply add/append a dummy export block (avoiding any complicated scripting logic to parse file) and the rest is taken care of by validation code. I have tested the logic when we have exports already present in ganesha.conf. 

> bash-4.2$ cortxfscli fs create fs4
> Created
> bash-4.2$ cortxfscli endpoint create fs4 proto=nfs,secType=sys,Filesystem_id=192.1,client=1,clients=*,Squash=no_root_squash,access_type=RW,protocols=4,pnfs_enabled=false,data_server=10.230.246.225
> Using embedded validation rules
> Created
> bash-4.2$ sudo /opt/seagate/cortx/cortx-fs-ganesha/bin/nfs_setup.sh restart                                                                                          
> 
> Stopping NFS Ganesha
> 
> Updating conf file
> 
> Starting NFS Ganesha
> bash-4.2$ sudo vim /etc/ganesha/ganesha.conf
> bash-4.2$ cortxfscli fs list
> OK
> FS ID   FS Name                             Exported        Protocol    Export Options
> 1       fs4                                 YES             nfs         {"proto": "nfs", "secType": "sys", "Filesystem_id": "192.1", "client": "1", "clients": "*", "Squash": "no_root_squash", "access_type": "RW", "protocols": "4", "pnfs_enabled": "false", "data_server": "10.230.246.225"}